recipes/nextsilicon-cpp-subprocess/all/conandata.yml

```diff
@@ -0,0 +1,7 @@
+sources:
+  "2.0":
+    url: "https://github.com/nextsilicon/cpp-subprocess/archive/refs/tags/v2.0.tar.gz"
+    sha256: "886df0a814a7bb7a3fdeead22f75400abd8d3235b81d05817bc8c1125eeebb8f"
+  "2.0.2":
+    url: "https://github.com/nextsilicon/cpp-subprocess/archive/refs/tags/v2.0.2.tar.gz"
```

Comment:
Why nextsilicon's fork and not arun11299? If arun11299 is abandoned, is nextsilicon the main fork now?

Reply:
Not sure if you read the previous thread.

I'm not 100% sure what's in the original project, but there are some PRs open for quite a time, and I didn't get any answer about my PRs, so we figured the best way to go is to fork it.

We clearly marked the fork in the package name.

Reply:
Okay, nextsilicon looks a good candidate: https://useful-forks.github.io/?repo=arun11299%2Fcpp-subprocess

Reply:
🤯 https://useful-forks.github.io/ is amazing!!!

Thanks for sharing 🎉